src/pages/Donate/DonateSupportUsRightRail.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+	<div class="donate-right-col row">
+		<div class="donate-right-col__meter small-12 medium-5 large-12 columns" v-if="showMeter">
+			<donation-meter :content="progressMeterEntry" />
+		</div>
+		<div class="donate-right-col__callouts small-12 medium-7 large-12 columns">
+			<ul>
+				<li v-for="(callout, index) in donationCallouts" :key="index">
+					<kv-contentful-img
+						v-if="callout.imageUrl"
+						class="donate-right-col__callouts--icon"
+						:contentful-src="callout.imageUrl"
+						:alt="callout.imageTitle"
+						fallback-format="png"
+						:width="49"
+						:height="48"
+					/>
+					<div class="donate-right-col__callouts--copy">
+						<h4 class="donate-right-col__callouts--header">
+							{{ callout.title }}
+						</h4>
+						<div class="donate-right-col__callouts--subhead" v-html="callout.subHead"></div>
+					</div>
+				</li>
+			</ul>
+		</div>
+	</div>
+</template>
+
+<script>
+import KvContentfulImg from '@/components/Kv/KvContentfulImg';
+import DonationMeter from '@/pages/Donate/DonationMeter';
+import { settingEnabled } from '@/util/settingsUtils';
+import { documentToHtmlString } from '~/@contentful/rich-text-html-renderer';
+
+export default {
+	components: {
+		DonationMeter,
+		KvContentfulImg,
+	},
+	inject: ['apollo', 'cookieStore'],
+	props: {
+		content: {
+			type: Object,
+			default: () => {}
+		}
+	},
+	data() {
+		return {};
+	},
+	computed: {
+		donationCallouts() {
+			const callouts = [];
+
+			const makeLoansHappenCopy = this.content?.contents?.find(
+				item => item.key === 'web-donate-support-us-make-loans-happen'
```

Comment:
I think I would prefer this key and the next key to be something a little more generic maybe `web-donate-support-us-callout-1` -- My thinking is that when this content changes the key will not be tied to the content

Reply:
Good idea!

Reply:
Updated